Gordon360/Services/RecIM/TeamService.cs

```diff
@@ -284,7 +289,7 @@ public async Task<TeamViewModel> PostTeamAsync(TeamUploadViewModel t, string use
                 Username = username,
                 RoleTypeID = 5
             };
-            await AddUserToTeamAsync(team.ID, captain);
+            await AddParticipantToTeamAsync("",team.ID, captain);
```

Comment:
It would be better to pass null here instead of an empty string.

Reply:
Can't because it's a parameter in AddParticipantToTeamAsync(string inviterUsername, int teamID, ParticipantUploadViewmodel participant)

Reply:
I could make it a default parameter if that is what you prefer